### ISSUE TITLE
[BUG] SC Controller registers random inputs while using Controller

### ISSUE THREAD

**Issue by TomCoursow** [kozec/sc-controller#655]
Hi

I tried playing Cemu with my Steam Controller using the "SC Controller" Tool...
While playing I noticed that if SC-Controller is running, the Controller is sending random inputs.
I can reproduce this best going crazy on the right Trackpad... But it does also happen with other things as well...

I made a short video for a better visualization using the Test Input Mode of SC-Controller.
Is this a known issue or are there workarounds for this already?

https://user-images.githubusercontent.com/17900090/147565804-a9388834-5f3f-4139-a9a0-ef06998642d1.mp4

**Additional Info:**
- Steam not running in background (but installed)
- Manjaro Linux - SC Controller installed via AUR
- Kernel: Linux 5.15.7-1
- Desktop Environment: Gnome 41
- Steam Controller directly connected via Mainboard-Bluetooth (no Steam Controller USB Dongle)

Best regards,
Tom

**Comment by kozec** [comment]
No, this is detinitelly not normal nor known issue. How sure you are it's not a HW issue? Does same thing happens when controller is connected using cable?

**Comment by TomCoursow** [comment]
Will try it with a cable in a moment. (Until then...)
- The issue does not happen if I use the Controller with Steam open (Desktop Configuration as Gamepad) but then I can not use the Gyro in Cemu (which is actually the only thing I want to achieve...) so I don't think that it is a Hardware Fault.
- Also if I setup a PS4 Controller using SC-Controller it does also not happen (but Gyro does not seem to be supported with PS4 Controller?)

**Comment by TomCoursow** [comment]
Ok tried connecting the Steam Controller via Cable.
Seems that using a cable it does not happen... weird...
So it is a Bluetooth issue or something?
